compiler/src/test/scala/dx/translator/ExtrasTest.scala

```diff
@@ -399,65 +399,56 @@ class ExtrasTest extends AnyFlatSpec with Matchers {
     val appId: String = dxApi.resolveApp("cloud_workstation").id
     val reorg: JsValue =
       s"""|{
-          | "custom_reorg" : {
+          |  "customReorgAttributes" : {
           |    "app_id" : "${appId}",
           |    "conf": null
           |  }
           |}
           |""".stripMargin.parseJson
 
-    val extras = extrasParser.parse(reorg)
+    val extras = Extras.parse(reorg)
     extras.customReorgAttributes shouldBe Some(CustomReorgSettings(appId))
   }
 
   it should "generate valid JSON execution policy" in {
     val expectedJs: JsValue =
       """|{
-         | "executionPolicy": {
-         |    "restartOn": {
-         |       "*": 5
-         |    },
-         |    "maxRestarts" : 4
-         | }
+         |  "restartOn": {
+         |    "*": 5
+         |  },
+         |  "maxRestarts" : 4
          |}
          |""".stripMargin.parseJson
 
-    val execPolicy = DxExecPolicy(Map("*" -> 5), Some(4))
-    JsObject(execPolicy.toJson) should be(expectedJs)
+    val execPolicy = DxExecPolicy(Some(Map("*" -> 5)), Some(4))
+    execPolicy.toJson shouldBe expectedJs
   }
 
   it should "generate valid JSON timeout policy" in {
     val expectedJs: JsValue =
       """|{
          |  "timeoutPolicy": {
-         |     "*": {
-         |        "hours": 12,
-         |        "minutes" : 30
-         |     }
+         |    "*": {
+         |      "hours": 12,
+         |      "minutes" : 30
+         |    }
          |  }
          |}
          |""".stripMargin.parseJson
-
-    val timeout = DxTimeout(None, Some(12), Some(30))
-    JsObject(timeout.toJson) should be(expectedJs)
+    val runSpec = DxRunSpec(None, None, None, Some(DxTimeout(None, Some(12), Some(30))))
+    runSpec.toJson shouldBe expectedJs
   }
 
   it should "handle default runtime attributes" in {
     val runtimeAttrs: JsValue =
       """|{
-         | "default_runtime_attributes" : {
-         |     "docker" : "quay.io/encode-dcc/atac-seq-pipeline:v1",
-         |     "zones": "us-west1-a us-west1-b us-west1-c us-central1-c us-central1-b",
```

Comment:
The behavior before was to ignore unrecognized attributes. This is bad behavior because the user might think an attribute they put there is having some effect. Now there is an error at compile time if there is an unknown attribute. There is already another test to make sure an error is thrown if there are invalid defaultRuntimeAttributes.